### PR TITLE
feat: update to detray v0.55.0

### PR DESCRIPTION
**Pull request by niermann999**
Update to detray v0.55.0 and add extra some includes

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,3 +33,5 @@
 
 # VSCode build output.
 out/
+data/
```
Comment (krasznaa) [inline .gitignore:36]: Nope. The `data/` directory must not be ignored. We continue to have the data download/creation scripts in there...

**Review by krasznaa — CHANGES_REQUESTED** [review]
The code updates I'm okay with. But the `.gitignore` change needs to be removed from this PR. If we need to change that file for some reason, let's do so in a separate PR.

File: .gitignore (hunk 1 above)
Comment (niermann999) [reply .gitignore:36]: Ah, oops, I do this locally everytime, since otherwise code searches in vscode take ages. Will revert

Comment (niermann999) [reply .gitignore]: Done

**Review by krasznaa — APPROVED** [review]
The code changes look okay as far as I can tell.

**Pull request merged**